tablo/models.py

```diff
@@ -1075,23 +1106,29 @@ def create_database_table(row_set, csv_info, dataset_id, append=False, additiona
 def add_geometry_column(dataset_id, is_import=True):
 
     with connection.cursor() as c:
-        add_command = (
-            "SELECT AddGeometryColumn ('{schema}', '{table_name}', '{column_name}', {srid}, '{type}', {dimension})"
-        ).format(
-            schema='public',
-            table_name=TABLE_NAME_PREFIX + dataset_id + (IMPORT_SUFFIX if is_import else ''),
-            column_name=GEOM_FIELD_NAME,
-            srid=WEB_MERCATOR_SRID,
-            type='GEOMETRY',
-            dimension=2
+        table_name = '{}{}{}'.format(TABLE_NAME_PREFIX, dataset_id, (IMPORT_SUFFIX if is_import else ''))
+        check_column_query = 'SELECT column_name FROM information_schema.columns WHERE table_name=\'{}\' and column_name=\'{}\''.format(
```

Comment:
I suggest wrapping in double quotes, instead of escaping singles:
```python
        check_column_query = "SELECT column_name FROM information_schema.columns WHERE table_name='{}' and column_name='{}'".format(
            table_name, GEOM_FIELD_NAME
        )
```

Reply:
Or you might consider as follows for readability:
```python
        check_column_query = '{select} {where}'.format(
            select='SELECT column_name FROM information_schema.columns',
            where="WHERE table_name='{}' and column_name='{}'".format(table_name, GEOM_FIELD_NAME)
        )
```

Reply:
I'll go with double quotes.